indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -150,6 +150,7 @@
     padding: 0;
     &.titled > .i-box-content {
         margin-top: 0;
+        text-align: center;
```

Comment:
And I'm not event sure we want to center the **empty** message. That was Jacques initial approach and it looked quite off.